client/lib/analytics/ad-tracking/setup.js

```diff
@@ -209,3 +144,72 @@ function setupWpcomFloodlightGtag() {
 
 	window.gtag( 'config', TRACKING_IDS.wpcomFloodlightGtag );
 }
+
+export const setupAllTrackingScripts = () => {
+	if ( typeof window !== 'undefined' ) {
+		if ( mayWeTrackByTracker( 'ga' ) ) {
+			setupGtag();
+		}
+
+		// Facebook
+		if ( mayWeTrackByTracker( 'facebook' ) ) {
+			setupFacebookGlobal();
+		}
+
+		// Bing
+		if ( mayWeTrackByTracker( 'bing' ) && ! window.uetq ) {
+			window.uetq = [];
+		}
+
+		// Criteo
+		if ( mayWeTrackByTracker( 'criteo' ) && ! window.criteo_q ) {
+			window.criteo_q = [];
+		}
+
+		// Quantcast
+		if ( mayWeTrackByTracker( 'quantcast' ) && ! window._qevents ) {
+			window._qevents = [];
+		}
+
+		// Google Ads Gtag for wordpress.com
+		if ( mayWeTrackByTracker( 'googleAds' ) ) {
+			setupWpcomGoogleAdsGtag();
+		}
+
+		if ( mayWeTrackByTracker( 'floodlight' ) ) {
+			setupWpcomFloodlightGtag();
+		}
+
+		// Twitter
+		if ( mayWeTrackByTracker( 'twitter' ) ) {
+			setupTwitterGlobal();
+		}
+
+		// Linkedin
+		if ( mayWeTrackByTracker( 'linkedin' ) ) {
+			setupLinkedinGlobal();
+		}
+
+		// Quora
+		if ( mayWeTrackByTracker( 'quora' ) ) {
+			setupQuoraGlobal();
+		}
+
+		// Outbrain
+		if ( mayWeTrackByTracker( 'outbrain' ) ) {
+			setupOutbrainGlobal();
+		}
+
+		// Pinterest
+		if ( mayWeTrackByTracker( 'pinterest' ) ) {
+			setupPinterestGlobal();
+		}
+
+		// AdRoll
+		if ( mayWeTrackByTracker( 'adroll' ) ) {
+			setupAdRollGlobal();
+		}
+	}
+};
+
+setupAllTrackingScripts();
```

Comment:
Did we really mean to call this here? If we did, why do we need to call it again in `loadTrackingScripts()`?

Reply:
I tried to remove that line, but then understood that we have code that depends on auto-invoked code on imported files (`import './setup';`)

The example is the `client/lib/analytics/ad-tracking/ad-track-registration.js` file. To change it is a bit invasive, and that's why I left it like this, for now, to keep backward compatibility - and do not increase the scope of this PR.

Reply:
If that's the case, then we shouldn't be exporting `setupAllTrackingScripts()` because someone might want to import it and might not notice that it's already running as a side effect.

Reply:
It is expected to execute this function multiple times. Before implementing bucketing, trackers were fired based on the static constant defined in the code. However, now, it can change during the runtime, e.g., before and after the cookie banner has been accepted by the user.